controllers/mongodb_tls_test.go

```diff
@@ -318,6 +318,17 @@ func TestAutomationConfigIsCorrectlyConfiguredWithTLS(t *testing.T) {
 		}
 	})
 
+	t.Run("With LogRotate and Systemlog enabled", func(t *testing.T) {
```

Comment:
Does it need a test case with syslog but without logrotate?

Reply:
After some thinking and reading the documentation, while it is not working, it is nothing I want the user to be blocked on. They should be informed but not blocked

Reply:
Perfect